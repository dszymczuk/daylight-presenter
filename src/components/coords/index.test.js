import React from "react";
import { render, cleanup } from '@testing-library/react';
import Coords from "./index";

afterEach(cleanup);

it("renders Coords", () => {
  const { asFragment } = render(<Coords lat={20} lng={30}/>);
  expect(asFragment()).toMatchSnapshot();
});