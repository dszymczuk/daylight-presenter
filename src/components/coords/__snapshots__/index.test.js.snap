// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Coords 1`] = `
<DocumentFragment>
  <span>
    (20, 30)
  </span>
</DocumentFragment>
`;
